src/test/java/org/gridsuite/directory/notification/server/DirectoryNotificationWebSocketHandlerTest.java

```diff
@@ -164,8 +169,6 @@ private void withFilters(String filterUpdateType) {
         }).collect(Collectors.toList());
         assertEquals(expected, actual);
         assertNotEquals(0, actual.size());
-        assertEquals(0, actual.stream().filter(m -> m.get(HEADER_DIRECTORY_UUID) != null && m.get(HEADER_DIRECTORY_UUID).equals("private_" + otherUserId)).count());
-        assertEquals(0, actual.stream().filter(m -> m.get(HEADER_DIRECTORY_UUID) != null && m.get(HEADER_DIRECTORY_UUID).equals("public_" + otherUserId) && m.get(HEADER_ERROR) != null).count());
```

Comment:
why did we remove those assertions ?

Reply:
because these assertions didn't bring anything really + it made the the function withFilter specific so not usable any other filter